.github/workflows/generate_calamari_weights_files.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           CARGO_TERM_COLOR: always
         run: |
           source ${HOME}/.cargo/env
-          CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --release --verbose --features=runtime-benchmarks
```

Comment:
There's one more place this is used - in the `publish_draft_release.yml` workflow to create the binary we release officially.

Reply:
> There's one more place this is used - in the `publish_draft_release.yml` workflow to create the binary we release officially.

Thanks! I forgot to `git add` it.